service/application/src/test/java/com/solace/maas/ep/event/management/agent/mop/MopTests.java

```diff
@@ -52,7 +52,7 @@ public void ack() {
 
     @Test
     public void testHeartbeatAgentMOPProtocol() {
-        HeartbeatMessage message = new HeartbeatMessage("runtimeId", "timestamp");
+        HeartbeatMessage message = new HeartbeatMessage("runtimeId", "timestamp", "1.1.2");
```

Comment:
Since this change is not backwardCompatible, we should upversion ema on the minor version